packages/ckeditor5-dev-release-tools/lib/release/updateversions.js

```diff
@@ -6,55 +6,70 @@
 'use strict';
 
 const { globSync } = require( 'glob' );
-const path = require( 'path' );
 const fs = require( 'fs-extra' );
 const semver = require( 'semver' );
-const getPackageJson = require( '../utils/getpackagejson' );
 const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
+const getPackageJson = require( '../utils/getpackagejson' );
+const normalizePath = require( '../utils/normalizepath' );
 
 /**
+ * Updates version of root package and subpackages if `packagesDirectory` path is provided.
+ *
  * @param {Object} options
  * @param {String} options.version
  * @param {String} [options.packagesDirectory]
  * @param {String} [options.cwd]
  */
 module.exports = function updateVersions( { packagesDirectory, version, cwd = process.cwd() } ) {
-	const packagesGlobs = packagesDirectory ? [ packagesDirectory + '/*', './' ] : './';
+	const normalizedPackagesDir = normalizePath( packagesDirectory );
```

Comment:
This line will throw an error if `packagesDirectory` is `undefined`. Your unit tests have a mocked version of `normalizePath()` so they don't crash in that case.